### PR TITLE
0.0.9 release

### PR DESCRIPTION
**Pull request by shammamah**
## About
This is the 0.0.9 release, and will be "officially" published on PyPI. The README has been updated and will now automatically be included in the PyPI page for the project. (See https://test.pypi.org/project/dash-bio/ -- note that this is the version of the README before I made corrections in 9734d5e).

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

**Comment by mkcor** [comment]
How about merging #336 first?

**Comment by shammamah** [comment]
@mkcor Looks like all of the tests are passing -- please take a look through the files changed in this PR and let me know what to change (if anything needs to be changed).

File: README.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
 # Dash Bio
 
-[![CircleCI](https://circleci.com/gh/plotly/dash-bio/tree/master.svg?style=svg)](https://circleci.com/gh/plotly/dash-bio)
```
Comment (mkcor) [inline README.md]: When did we decide to remove the CI status badge?

File: README.md [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
-Demo: https://dash-bio.plotly.host/Portal/
+## Components
 
-## Progress
+The Dash Bio components each fall into one of three categories:
 
-#### Bioinformatics charts
+- Custom chart types
+- Sequence analysis tools
+- Visualization tools
```
Comment (mkcor) [inline README.md]: The third category is extremely general and includes the first two categories, so I don't think that division makes a lot of sense... Where does it come from?

Comment (mkcor) [inline README.md]:
```suggestion
- 3D rendering tools
```
Ok, based on what "Visualization tools" includes, I could see this as more fitting...

File: README.md [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
-   ```
-2. Run `python index.py`
-3. Visit http://localhost:8050 in your web browser and click on the component
-   you want to explore.
+## Dash
+
+Learn more about Dash at
+[https://plot.ly/products/dash/](https://plot.ly/products/dash/).
+
+## Consulting and OEM
```
Comment (mkcor) [inline README.md]: What's OEM?

File: README.md [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
+
+For inquiries about Dash app development, advanced OEM integration,
+and more, please visit [this
```
Comment (mkcor) [inline README.md]:
```suggestion
and more, please [reach
```

File: README.md [hunk 5]
```diff
@@ -1,55 +1,55 @@
...
+link](https://plotly.typeform.com/to/mH1Cpb).
```
Comment (mkcor) [inline README.md]:
```suggestion
out](https://plotly.typeform.com/to/mH1Cpb).
```

File: README.md (hunk 1 above)
Comment (mkcor) [reply README.md]: We may want to move it right next to the level-1 "Dash Bio" title, it's quite conventional, or further down in the README, if you prefer.

File: README.md (hunk 3 above)
Comment (shammamah) [reply README.md]: https://en.wikipedia.org/wiki/Original_equipment_manufacturer
https://www.lifewire.com/what-is-oem-software-1701736

Referred to on the Plotly site here:
https://plot.ly/products/consulting-and-oem/

File: README.md (hunk 1 above)
Comment (shammamah) [reply README.md]: Resolved in 1babfd7.

Comment (mkcor) [reply README.md]: Ok, but the line added in 1babfd7 is prefixed with two whitespaces.

File: README.md (hunk 3 above)
Comment (mkcor) [reply README.md]: Ok, thanks, "Consulting and OEM" makes sense then ("inquire about an OEM" less so).

File: README.md (hunk 1 above)
Comment (shammamah) [reply README.md]: Fixed in ea4567a.

**Pull request merged**